src/integrationTest/java/uk/gov/hmcts/darts/common/controller/AuditApiTest.java

```diff
@@ -113,8 +113,8 @@ void searchForAuditWithWrongParams() throws Exception {
         MockHttpServletRequestBuilder requestBuilder = get("/audit/search")
             .queryParam("wrong_query_param", "value")
             .contentType(MediaType.APPLICATION_JSON_VALUE);
-        mockMvc.perform(requestBuilder).andExpect(status().isBadRequest())
-            .andExpect(jsonPath("$.message", is("All filters were empty. ")));
+        mockMvc.perform(requestBuilder).andExpect(status().isInternalServerError())
```

Comment:
This status is wrong, it is not an internal server error, it needs to remain as a bad request.
i think we will need to create an AuditApiError class and define a couple of errors in it, and get AuditSearchQueryValidator to raise these errors